tools/misc/mapping_expander.py

```
@@ -0,0 +1,194 @@
+source_sheet_name = "mappings"           # Sheet name in the source file
+reference_sheet_name = "target"          # Sheet name in the reference file
+reference_column_name = "node_id"        # Column name in the reference file to match
+
+framework_exception = "soc2_rev2022"     # Set to None if no exceptions needed
+
+
+# === Load exceptions from JSON ===
+try:
+    with open(exceptions_json_path, "r", encoding="utf-8") as f:
+        exception_map = json.load(f)
+        print(f"ℹ️  Exceptions list for \"{framework_exception}\" loaded from JSON file \"{os.path.basename(exceptions_json_path)}\"")
```

Pylint (3.3.7)

[convention] Line too long (134/100)

(C0301)